competency-insights-ui/src/routes/AppRouters.js

```diff
@@ -1,12 +1,18 @@
-import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import { Routes, Route } from "react-router-dom";
+
+// import Dashboard from "../auth/Dashboard";
+
 import {Login, ProfilePage} from "../pages";
 export const AppRouters = () => {
     return (
-        <Router>
-            <Routes>
+        <Routes>
+            <Route>
                 <Route path="/" element={<Login />} />
+                {/* <Route path="/dashboard" element={<Dashboard />} /> */}
```

Comment:
The dashboard page and route are not ready yet. if keep this routing is not breaking the functionality then we can keep route active i.e. uncomment it.

Reply:
It is giving runtime error as there is no such file found.